library/pod-security-policy/capabilities/mutation.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: Assign
+metadata:
+  name: k8spspcapabilities
+spec:
+  applyTo:
+  - groups: [""]
+    kinds: ["Pod"]
+    versions: ["v1"]
+  location: "spec.containers[name:*].securityContext.capabilities.add"
+  parameters:
+    assign:
+      value: ["NEW_CAPABILITY"] # default add capability
```

Comment:
is `NEW_CAPABILITY` an actual thing? If these are toy examples, we should be careful where we put them.

Reply:
Yes, it's just an example. I was hesitant on adding an actual capability, just in case someone adds it without checking.